### DISCUSSION TITLE
Could we have a forum for people new to networking wanting to use security onion?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5698] [Unsupported Versions] [answered]
Hi, i know security onion dosnt work withhout networking. I also know that folks at security onion are very busy and dont have time to take questions about basic networking. 
  It seems like questions about certain parts of security onion bleed over into basic networking, especially for the novice or newbie that "bleed over" might not be readily understood. 
  But if an extra security onion forum discussion link could be created, then perhaps we could get our answers from other security onion forum users? 
  This could also help rookies like myself be able to search for basic security onion concepts (not that there is a lack of security onion documentation out there). 
   I believe there might have been a couple of people that have had problems with installing security onion (especially those who dont have a lot of networking experience) that have simply given up. 
   Not everyone is like a bulldog with a bone like me, I feel like if someone else can do something, then so can i ....... evenly eventually. 
   If this new forum is possible, I would love to help with it. This could be a way from me and others to learn?
   Just a thought, thanks for any consideration in letting me know what you think.

**Comment by dougburks** [comment, marked as answer]
I've suggested to you many times now that basic networking knowledge is a prerequisite for cybersecurity in general and that basic networking questions are beyond the scope of this discussions forum:
https://github.com/Security-Onion-Solutions/securityonion/discussions/5131
https://github.com/Security-Onion-Solutions/securityonion/discussions/5134
https://github.com/Security-Onion-Solutions/securityonion/discussions/5652

There are many sites on the Internet that are specifically designed to help you learn basic networking knowledge. However, this discussions site is not one of them.

**Reply by iqworks** [reply]
Yes you have mentioned that many times and I have begun looking elsewhere for further learning. I apologize for overlapping questions about basic networking with security onion. 
Those lines are easily crossed for people new to both. I don’t want to be a nuisance.  Please ignore any questions that I pose that have to do with basic networking. I still need to 
Use this forum because I think security onion is the best place to start to begin using these tools. 
Thanks for you advice, suggestions and patience.
